js/utils/templateHelpers.js

```diff
@@ -1,5 +1,10 @@
-// todo: add in some helpers!
+import app from '../app';
+
+const polyT = function (...args) {
+  return app.polyglot.t(...args);
+};
+
+export {
```

Comment:
LOL. That's how I had it before, I changed it to this based on our conversation earlier about switching to this pattern.